Remove wifi driver

francois-berder-imgtec opened the pull request:
This list of patches completely removes the wifi driver from openwrt. This pull request should be merged before https://github.com/CreatorDev/openwrt/pull/53 is merged. Once both pr are merged, the wifi driver will be added back by merging https://github.com/CreatorDev/Ci40-platform-feed/pull/102.

francois-berder committed:
Remove uccp420wlan kernel package definition

francois-berder committed:
Remove wifi firmware

francois-berder committed:
mac80211: Add patch to avoid compilation

The wifi driver (uccp420wlan) of the Ci40 contains many reference to
ieee80211_band. Adding it to mac80211 avoids changing it in the driver.

Signed-off-by: Francois Berder <francois.berder@imgtec.com>

abhijit-mahajani-imgtec reviewed (changes requested):
> > Remove CDRA database from kernel

fix typo

ghost commented:
@abhijit-mahajani-imgtec I removed the patch concerning CRDA.

abhijit-mahajani-imgtec reviewed (changes requested):
why not to remove whole firmware directory?

abhijit-mahajani-imgtec commented:
If we removing wifi driver for the time being, what about wifi configurations listed in /etc/uci-defaults/config/network and wireless file itself?

abhijit-mahajani-imgtec commented on target/linux/pistachio/modules.mk:
```diff
@@ -192,48 +192,3 @@ define KernelPackage/nhc_fragment/description
 endef
 
 $(eval $(call KernelPackage,nhc_udp))
```
In stead of removing whole definition from modules.mk, how about just removing kmod-uccp420wlan and kmod-cfg80211 from target.mk. I think with that you won't need to add the patches of mac80211 too..

francois-berder-imgtec replied:
We will not need anymore the definition in modules.mk once the wifi driver is out of the kernel since it will be in Ci40-platform-feed.
I should remove kmod-uccp420wlan and kmod-cfg80211 from target.mk for now.
The patch for mac80211 could be removed but it avoids doing a lot of changes in the wifi driver, that is why I preferred modifying mac80211.

ghost commented:
> why not to remove whole firmware directory?

I think I did that no ?

> If we removing wifi driver for the time being, what about wifi configurations listed in /etc/uci-defaults/config/network and wireless file itself?

Are we going to use it these files later once the wifi driver is out of the kernel ? If yes, I think we should leave it there.

francois-berder committed:
Remove wifi driver and cfg80211 from default package list

The wifi driver is removed since there is no definition of the package.
It will have to be added back once the Ci40-platform-feed will contain
the wifi driver.

Signed-off-by: Francois Berder <francois.berder@imgtec.com>

abhijit-mahajani-imgtec commented:
`

> why not to remove whole firmware directory?
> I think I did that no ?
> 
> If we removing wifi driver for the time being, what about wifi configurations listed in /etc/uci-defaults/config/network and wireless file itself?
> Are we going to use it these files later once the wifi driver is out of the kernel ? If yes, I think we should leave it there.

`

Ok.

abhijit-mahajani-imgtec commented:
Verified that now wifi won't work since its not enabled and cannot build uccp420wlan
